data/make_user_link.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+USER_UID=$(loginctl list-sessions | grep seat0 | tr -s " " | cut -d " " -f 3)
+ln -fs /run/user/$USER_UID /run/currentuser
```

Comment:
Should this be part of sessions package?

Reply:
Current or active ?

Systemd wise it should be matched to the active user.

Reply:
Link creation moved jolla-sessions-qt5 and renamed to activeuser

Reply:
Ok